functions.php

```diff
@@ -515,3 +515,60 @@ function twentytwenty_read_more_tag() {
 	add_filter( 'the_content_more_link', 'twentytwenty_read_more_tag' );
 
 }
+
+/**
+ * WIP: These should be moved to the Custom class.
+ */
+add_action( 'customize_controls_enqueue_scripts', function() {
+
+	// Add script for color calculations.
+	wp_enqueue_script(
+		'twentytwenty-color',
+		get_template_directory_uri() . '/assets/js/color.js',
+		[ 'wp-color-picker' ],
+		time(), // WIP: Using time() instead of a real version for cache-busting while developing.
+		false
+	);
+
+	// Add script for controls.
+	wp_enqueue_script(
+		'twentytwenty-customizer',
+		get_template_directory_uri() . '/assets/js/customizer.js',
+		[ 'twentytwenty-color', 'customize-controls', 'underscore', 'jquery' ],
+		time(), // WIP: Using time() instead of a real version for cache-busting while developing.
+		false
+	);
+});
```

Comment:
These will need to be moved, do we want them in the customize class? Or do we just convert them to normal functions and leave them in functions.php?

And of course we need to change the version to a normal version instead of `time()`, that's just to save some time when developing and changing scripts constantly to avoid cache issues in the browser.